gamemode/shared.lua

```diff
@@ -5,6 +5,8 @@ GM.Website = "https://github.com/cfc-servers/raft_v_raft"
 RVR = RVR or {}
 
 AddCSLuaFile( "load_config.lua" )
+AddCSLuaFile( "localization.lua" )
```

Comment:
nitpicking, but I'm pretty sure this should be `Localisation` not `Localization`

Reply:
I think both are valid spellings

Reply:
`Localization` is typically the US spelling; I specifically went with it 'cause the englishman

Reply:
I'm very upset